scripts/travis/before_build.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-yarn global add grunt-cli
+yarn global add grunt-cli@1.2.0
```

Comment:
AFAICT, this will use the latest 1.x release. Is that intended or should we use the [`--exact` option][1]?

[1]: https://yarnpkg.com/en/docs/cli/add#toc-yarn-add-exact-e